bundles/org.eclipse.swt/Eclipse SWT/gtk/org/eclipse/swt/widgets/Display.java

```diff
@@ -2349,6 +2401,19 @@ public static Display getDefault () {
 	}
 }
 
+/**
+ * Returns an executor that will execute any submitted runnable in the user-interface thread, regardless of the calling thread
+ * @return an {@link Executor} that executes in the user-interface thread.
+ * @since 3.120
+ */
+public static Executor executor() {
```

Comment:
Since this is static it could be better placed in a separate class to avoid further bloat of Display - and in a platform independent class.

Reply:
People are more used to use the Display class and I don't like introduce more "helper" classes just to hide the SWT bloat.